src/bp/core/converse/converse-router.ts

```diff
@@ -89,6 +96,22 @@ export class ConverseRouter extends CustomRouter {
     )
   }
```

Comment:
Instead of a comment, can you move methods into "setupAuthenticatedRoutes" and "setupUnauthenticatedRoutes" ?